test/functional/cmdline_options_testresources/src_90/com/ibm/j9/getcallerclass/ReflectionMHTests.java

```diff
@@ -62,7 +62,7 @@ public static boolean test_getCallerClass_Helper_Reflection_fromBootExtWithAnnot
 			cls = (Class<?>) method.invoke(null, new Object[0]);
 
 			boolean isClassNameExpected;
-			if (VersionCheck.major() >= 18) {
+			if (VersionCheck.major() >= 23 || (VersionCheck.major() >=18 && Boolean.getBoolean("jdk.reflect.useDirectMethodHandle"))) {
```

Comment:
Missing space after `>=`, missing parentheses around both `>=` subexpressions.

Reply:
Fixed in https://github.com/eclipse-openj9/openj9/compare/3ca1654a15b61c437ff36bb8c5274a43d38fbb11..57a5ddfa22038e022ce188f6c7e5374099639523